package.json

```diff
@@ -144,6 +144,12 @@
       ],
       "src/*": [
         "src/*"
+      ],
+      "test/playwright": [
+        "e2e/setup/index.ts"
+      ],
+      "test/jest": [
+        "jest.setup.ts"
```

Comment:
now consumers can use it

Reply:
but I don't knowe how to make typescipt auto complete `fabric/tes` => `fabric/test/..`